src/platforms/common/configuration/releases.mdx

```diff
@@ -88,8 +88,12 @@ In order to monitor release health, the SDK sends session data.
 
 A session represents the interaction between the user and the application. Sessions contain a timestamp, a duration, a status (if the session was OK or if it crashed), and are always linked to a release. Most Sentry SDKs can manage sessions automatically.
 
+<PlatformSection notSupported={["rust"]}>
```

Comment:
we do have this in rust: https://docs.rs/sentry/latest/sentry/struct.ClientOptions.html#structfield.auto_session_tracking